scripts/contact.js

```diff
@@ -0,0 +1,81 @@
+const form = document.getElementById('form');
+const name = document.getElementById('name');
+const email = document.getElementById('email');
+const question = document.getElementById('question');
+
+form.addEventListener('submit', e => {
+	e.preventDefault();
+	checkInputs();
+});
+
+function checkInputs(){
+    // get the values from the inputs
+    const nameValue = name.value.trim();
+    const emailValue = email.value.trim();
+    const questionValue = question.value.trim();
+
+    if(nameValue === ''){
+        //show error and add error class
+        setErrorFor(name,"Name cannot be blank");
+    } else {
+        //add success class
+        setSuccessFor(name);
+    }
```

Comment:
You might also wanna check if the name is a number or not because realistically a person won't have a numerical name

Reply:
done. I added this condition into my code